docs/user/02-logs.md

```diff
@@ -473,3 +473,22 @@ Each Fluent Bit Pod can process up to 10 MB/s of logs for a single LogPipeline.
 ### Max Amount of Pipelines
 
 The maximum amount of LogPipelines is 5.
+
+## Troubleshooting
+
+### Logs Not Arriving at the Destination
+
+   Cause: The backend is not reachable or wrong authentication credentials are used.
```

Comment:
Maybe let's add the aspect of using a wrong endpoint. Saying "not reachable" sounds like a temporary outage.